internal/cmd/cmd.go

```diff
@@ -340,6 +340,7 @@ var runConnector = connector.Run
 func defaultConnectorOptions() connector.Options {
 	return connector.Options{
 		Addr: connector.ListenerOptions{
+			HTTP:    ":80",
```

Comment:
Should we allow HTTP listeners from the connector? Seems like we wouldnt want that.

Reply:
The HTTP service serves only a health endpoint. This is useful in the common case of external load balancers in public cloud where the load balancer health check often defaults to TCP checks which produces EOF errors when sent to the HTTPS service. By exposing a HTTP endpoint, the connector won't complain when the external load balancer hits it with a TCP handshake. It does not serve the reverse proxy so there's no security risk; sending a request path other than `/healthz` will return 404